indexes/zk_ee/ethereum-general-state-tests.yaml

```diff
@@ -6281,21 +6282,24 @@ entries:
         enabled: true
       static_Call1024PreCalls.json:
         hash: '0x89e8811d9d36b8e86f4c728bac00e719'
-        enabled: true
+        enabled: false
```

Comment:
Why?

Reply:
Thanks, forgot to add the comment (u64 overflow for ergs)